x/farming/spec/02_state.md

```diff
@@ -227,7 +230,7 @@ An example of `FixedAmountPlan`
 }
 ```
 
-An example of `RatioPlan`
+An example of `RatioPlan`:
```

Comment:
so the two plan types both use the same id and type? (surprised!)

I was expecting that at least the type would be different to distinguish between FixedAmountPlan and RatioPlan

`"id": 1,`

`"type": 1,`

Reply:
`FixedAmountPlan` and `RatioPlan` are distinguished by `epoch_amount` and `epoch_ratio`. `id` should be different if there are two plans in the network. It is automatically generated when a user sends either `FixedAmountPlan` or `RatioPlan`. `type` is about whether or not the plan is public or private.

https://github.com/tendermint/farming/blob/d34e62438dbfa4c6779f02e40af5d15b19e87155/x/farming/types/farming.pb.go#L36-L43